charts/celeborn/README.md

```diff
@@ -40,17 +43,57 @@ When you want to test the template rendering, but not actually install anything.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` | Pod affinity |
+| cluster.name | string | `"cluster"` | Specifies Kubernetes cluster name |
+| dnsPolicy | string | `"ClusterFirst"` | Specifies the DNS policy for Celeborn pods to use |
+| fullnameOverride | string | `""` | String to override the default generated fullname |
+| hostNetwork | bool | `false` | Specifies whether to use the host's network namespace |
+| image.pullPolicy | string | `"Always"` | Image pull policy |
+| image.pullSecrets | list | `[]` | Image pull secrets for private image registry |
+| image.repository | string | `"aliyunemr/remote-shuffle-service"` | Image repository |
+| image.tag | string | `"0.1.1-6badd20"` | Image tag |
+| nameOverride | string | `""` | String to override the default generated name |
+| nodeSelector | object | `{}` | Pod node selector |
+| podAnnotations | object | `{}` | Pod annotations |
+| podMonitor.enable | bool | `true` | Specifies whether a PodMonitor should be created |
+| podMonitor.podMetricsEndpoint | object | `{"interval":"5s","portName":"metrics","scheme":"http"}` | Specifies PodMonitor endpoint |
+| priorityClass.master.create | bool | `false` | Specifies whether a new priority class for Celeborn master pods should be created |
+| priorityClass.master.name | string | `""` | Specifies the name of priority class for Celeborn master pods to be used (created if `create: true`) |
+| priorityClass.master.value | int | `1000000000` | Specifies the integer value of this priority class, default is half of system-cluster-critical |
+| priorityClass.worker.create | bool | `false` | Specifies whether a new priority class for Celeborn worker pods should be created |
+| priorityClass.worker.name | string | `""` | Specifies the name of priority class for Celeborn worker pods to be used (created if `create: true`) |
+| priorityClass.worker.value | int | `999999000` | Specifies the integer value of this priority class, default is Celeborn master value minus 1000 |
+| replicaCount.master | int | `3` | Specifies the number of Celeborn master replicas to deploy, master replicas should not less than 3 |
+| replicaCount.worker | int | `5` | Specifies the number of Celeborn worker replicas to deploy, should less than node number |
+| resources.master | object | `{}` | Celeborn master pod resources |
+| resources.worker | object | `{}` | Celeborn worker pod resources |
+| securityContext.fsGroup | int | `10006` | Specifies the group ID to use when modifying ownership and permissions of the mounted volumes |
+| securityContext.runAsGroup | int | `10006` | Specifies the group ID to run the entrypoint of the container process |
+| securityContext.runAsUser | int | `10006` | Specifies the user ID to run the entrypoint of the container process |
+| service.port | int | `9097` | Specifies service port |
```

Comment:
Could this port distingish the port of the Master and Worker service?

Reply:
> Could this port distingish the port of the Master and Worker service?

Actually, `service.port` is used only by the master service, the worker service does not expose a service port.

Reply:
@ChenYi015, could the worker service expose a service port?

Reply:
@SteNicholas Just wondering, why do we need to expose a port for the worker service?

Reply:
@ChenYi015, could the users deploy worker service in other k8s cluster?

Reply:
@SteNicholas Currently not, we need to modify the helm chart so that we can deploy celeborn workers in another k8s cluster.

Reply:
@ChenYi015, would you like to support this in another pull request?

Reply:
@SteNicholas If that's you need, I will try to make it happen.